transports/sqs/configuration-options_visibility-timeout_sqs_[,7.3).partial.md

```diff
@@ -0,0 +1,11 @@
+## Message visibility
+
+The message visibility timeout in Amazon SQS defines how long a message remains hidden from other consumers after being received. If a handler takes longer to process a message than the configured visibility timeout, the message becomes visible again and may be reprocessed.
+
+Without a message visibility extension mechanism (supported in version 7.3 and above), this can result in:
+
+- Duplicate message processing, wasting resources
+- Message deletion failures, causing the message to reappear on the queue
+- Inaccurate delivery attempt counters, impacting poison message detection
+
+To avoid these issues, the visibility timeout should be set to at least match the maximum expected processing time of a message.
```

Comment:
The only option is to set it at the queue level or by enabling the hybrid mode, which comes with performance issues.

```suggestion
To avoid these issues, configure the visibility timeout at the queue level, when creating, to at least match the maximum expected processing time of a message.
```